### DISCUSSION TITLE
How do I switch from character-level to token-level operation?

### DISCUSSION THREAD

**Discussion by dratman** [karpathy/llm.c#204] [Q&A] [answered]
I have tried both Andrej's character-level Tiny Shakespeare and his token-level TinyStories. Does the same repo encompass both methods now? If so, how do I option for characters vs tokens?
Thx

**Comment by azret** [comment, marked as answer]
See https://github.com/karpathy/llm.c/pull/156

**Reply by dratman** [reply]
Ok, thanks. I get it now. There are two different build instructions.
